src/services/services.ts

```diff
@@ -1218,6 +1218,7 @@ namespace ts {
         InsertSpaceAfterFunctionKeywordForAnonymousFunctions: boolean;
         InsertSpaceAfterOpeningAndBeforeClosingNonemptyParenthesis: boolean;
         InsertSpaceAfterOpeningAndBeforeClosingNonemptyBrackets: boolean;

```

Comment:
Not that this should necessarily be a part of this change, but we should separately consider changing `Nonempty` to `NonEmpty`